src/lister_haxe/lister.hx

```diff
@@ -0,0 +1,38 @@
+import haxe.io.Path;
+using StringTools;
```

Comment:
`import` and `using` keywords: This was confusing and I still I don't totally feel like I fully understand the difference.  `import` takes a local or standard library and allows it to be referenced by code in the current file (`module`, in Haxe parlance).  `using` lets you pull in [extension methods](http://haxe.org/manual/lf-static-extension.html) onto existing, known types, but it's also how you use some of the other bits of the standard libraries (which are extensions instead of libraries).